dask_expr/reductions.py

```diff
@@ -176,7 +176,19 @@ def chunk_kwargs(self):
         return {"ignore_index": self.ignore_index, **self._subset_kwargs()}
 
     def _simplify_up(self, parent):
-        return
+        if self.subset is not None and isinstance(parent, Projection):
+            columns = parent.columns
+            if not isinstance(columns, list) and not hasattr(columns, "dtype"):
+                columns = [columns]
```

Comment:
No, that would change behavior. Everything that’s not a scalar would return a DataFrame instead of a series 

df[[a]] gives you back a frame instead of a Series

Reply:
Hrm, indeed

```python
In [1]: import dask_expr as dx

In [2]: df = dx.datasets.timeseries()

In [3]: df.x.expr.columns
Out[3]: 'x'

In [4]: df.expr.columns
Out[4]: Index(['name', 'id', 'x', 'y'], dtype='object')
```

I wonder if we want to get cleaner here in some way.  Using `.columns` as a signal for "this is a series" seems like we're mixing a couple of things into one variable.  Maybe expr.columns should always be a pandas index (or even just a list).  

@rjzamora any thoughts?

Reply:
Right, I remember struggling with this a bit. From the collection API, we want the behavior of `columns` to be consistent with that of a pandas object. However, for the `Expr`, it seems reasonable to always return a `list` (it could also simplify column-projection logic if we always know that `Expr.columns` will return a list).